templates/thank-you.html

```diff
@@ -16,20 +15,62 @@ <h1 data-qa="title">{{ _("Submission successful") }}</h1>
     "classes": "u-mb-s"
     })
   %}
+    <h1 data-qa="title">{{ _("Thank you for submitting the {survey_title}").format(
+      survey_title = survey_title)}}</h1>
     <p data-qa="guidance">
-      {% if content.trad_as %}
-        {{ _("Your answers were submitted for <span>{ru_name}</span> ({trading_as_name}) on {submitted_date_time}").format(
+      {% if survey_type == "social" %}
```

Comment:
We should avoid large conditionals in templates. We could make `thank-you.html` a common base template that defines the page structure and places where content can be overridden, and have templates for business/social etc that define the contents.

Reply:
Logic moved to backend now as discussed, thanks!